docs/workforce-management-framework/topics/pro-installing-core-dependencies.adoc

```diff
@@ -0,0 +1,20 @@
+[id='pro-installing-core-dependencies-{chapter}']
```

Comment:
Fair point - I'm putting it all back into one procedure and will avoid the extreme "breaking-up" of work